tests/fixtures.py

```diff
@@ -1,51 +1,50 @@
 import pytest
 from tests import helpers
-from ocs_ci.ocs import constants, defaults
-from ocs_ci.utility import templating
+from ocs_ci.ocs import constants
 
 
 @pytest.fixture()
 def create_rbd_secret(request):
     """
-    Create a rbd secret
+    Create an RBD secret
     """
     class_instance = request.node.cls
 
     def finalizer():
         """
-        Delete the project
+        Delete the RBD secret
         """
-        if class_instance.rbd_secret_obj.get():
-            class_instance.rbd_secret_obj.delete()
+        if class_instance.secret_obj.get():
```

Comment:
@ebenahar what is the reason for removing "rbd" from rbd_secret_obj ? 

wouldn't it be be better to have "distinct names for rbd and cephfs based secret_obj as most of the test cases are using these object names while using fixture & create_storageclass. In some test cases, we need both as distinct secret objs.

e.g.
   if interface_type == constants.CEPHBLOCKPOOL:
            secret_name = self.rbd_secret_obj.name
            interface_name = self.cbp_obj.name

        elif interface_type == constants.CEPHFILESYSTEM:
            secret_name = self.cephfs_secret_obj.name
            interface_name = helpers.get_cephfs_data_pool_name()

Reply:
@nehaberry will change it back